src/helpers/clipboard/clipboard_helpers.ts

```diff
@@ -54,3 +61,20 @@ export function getPasteZones<T>(target: Zone[], content: T[][]): Zone[] {
     height = content.length;
   return target.map((t) => splitZoneForPaste(t, width, height)).flat();
 }
+
+export function gePastablePluginClipBoardContent(
+  content: OSClipboardContent
+): ImportClipboardContent {
+  const htmlDocument = new DOMParser().parseFromString(
+    content[ClipboardMIMEType.Html] ?? "<div></div>",
+    "text/html"
+  );
+  const oSheetClipboardData = htmlDocument
+    .querySelector("div")
+    ?.getAttribute("data-osheet-clipboard");
```

Comment:
Just curious: are there any pro/cons of parsing the HTML rather than using a regex ? And of putting the data in an attribute rather than a HTML comment ?